usage/camp/src/test/java/org/apache/brooklyn/camp/brooklyn/DslAndRebindYamlTest.java

```diff
@@ -124,6 +126,19 @@ public void testDslAttributeWhenReady() throws Exception {
     }
 
     @Test
+    public void testDslAttributeWhenReadyPersisted() throws Exception {
```

Comment:
For the `java.io.NotSerializableException`: that's surprising. We shouldn't be relying on normal java serialization. We use xstream for persistence, which doesn't need `Serializable`.

Some test assertions can enable a stronger assertion about being able to serialize all of the persisted state using java serialization (in an attempt to fail-fast). We can just disable that, if it's being done. It's too strict.

Worth looking down the stacktrace to see what is trying to do the java serialization, to see if it is the test assertion or something else.